static/files/template-component-reference.md

```diff
@@ -0,0 +1,66 @@
+---
+linkTitle: <Component Name>
+title: <Component Name> Component
+description: The <Component Name>, which does <function>, is provided by the <Gem name> in Open 3D Engine (O3DE). # Example
+weight: 200 # Example
+toc: true
+---
+
+<!-- Introduction - Describe the component and what it does. -->
+
+
+## Provider
+
+[Gem's name](/docs/user-guide/gems/reference/<path-to-gem-docs>)
+
+
+## Dependencies
+
+<!-- (Optional) List the component's depdencies -->
+
+
+## Use cases
+
+<!-- (Optional) Write the use cases in which a user may want to use this component. -->
+
+
+## Limitations
+
+<!-- (Optional) Write the limitations of this component. Don't include any "future work". -->
+
+
+## Properties
+
+![<component name> interface](/images/<path-to-images>)
+
+| Property | Description | Value | default |
+| - | - | - | - |
+|   |   |   |   |
+|   |   |   |   |
+|   |   |   |   |
+
+
+## <Component feature>
+
+<!-- (Optional) Explain specific features of your component. Repeat this section for each additional topic, as needed. -->
+
+
+## <EBus interface name>RequestBus
+
+<!-- (Optional) Example: "Use the following request functions with the <Request EBus name> EBus interface to communicate with <components> in your game." -->
+
+| Request Name | Description | Paramter | Return | Scriptable |
```

Comment:
I've seen both "Request Name" and "Method Name" here. I prefer the more generic "Method Name", as seen in your Vegetation component example.
```suggestion
| Method Name | Description | Parameter | Return | Scriptable |
```